assets/js/pages/ClusterDetails/AttributeDetails.test.jsx

```diff
@@ -100,10 +100,36 @@ describe('AttributesDetails', () => {
     },
   ];
 
+  const assertAttributesDisplayed = (attributes) => {
+    Object.entries(attributes).forEach(([key, value]) => {
+      expect(screen.getByText(key)).toBeInTheDocument();
+      expect(screen.getByText(value)).toBeInTheDocument();
+    });
+  };
+
+  const assertResourcesDisplayed = (resources, expectedResource) => {
+    expectedResource.forEach(({ key, title }) => {
+      expect(screen.getByText(title)).toBeInTheDocument();
+      resources.forEach((resource) => {
+        let value;
```

Comment:
I put a let instead of const, as otherwise eslint complains about nested ternary operator. Maybe there is a better way ?